src/test/analyzer_test.ts

```diff
@@ -84,6 +85,32 @@ suite('Analyzer', () => {
           ['MyNamespace.SubBehavior', 'MyNamespace.SimpleBehavior']);
     });
 
+    // TODO(fks) 09-28-2016: Fix off-by-one error in warning sourceRange line
+    // numbers.
+    test('throws when cant find behavior', async() => {
```

Comment:
👍 that was my understanding as well but I wanted to double check that I wasn't missing some easier way.